src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs

```diff
@@ -5368,19 +5322,6 @@ public static void DisableFilter(ref ActivityFilter filterList, EventSource sour
         }
 
         /// <summary>
-        /// Returns the first ActivityFilter from 'filterList' corresponding to 'source'.
-        /// </summary>
-        public static ActivityFilter GetFilter(ActivityFilter filterList, EventSource source)
-        {
-            for (var af = filterList; af != null; af = af.m_next)
-            {
-                if (af.m_providerGuid == source.Guid && af.m_samplingFreq != -1)
-                    return af;
-            }
-            return null;
-        }
-
-        /// <summary>
         /// Returns a session mask representing all sessions in which the activity 
         /// associated with the current thread is allowed  through the activity filter. 
         /// If 'triggeringEvent' is true the event MAY be a triggering event. Ideally 
```

Comment:
To the extent possible, I'd rather have fewer ifdefs in this code simply because we'd like EventSource to be as close to the same as possible on all platforms.  Are there particular methods that you're concerned about?  I believe that all of the WriteEvent methods should be available on all platforms and should not get removed by the rewriter.  Is this not the case?